testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/deployment/structure/war/symbolicLink/SymlinkingUnitTestCase.java

```diff
@@ -62,6 +66,11 @@ public static void afterClass() throws IOException {
         controllerClient = null;
     }
 
+    @Before
+    public void before() {
+        Assume.assumeFalse(isWindows());
```

Comment:
you need to move this into beforeClass as otherwise test tries to create link on windows and fails